.github/workflows/main_workflow.yml

```diff
@@ -210,7 +210,7 @@ jobs:
 
           # build required images
           skaffold build --push=false
-          skaffold build -p release --push=false
+          skaffold build -p ci --push=false
```

Comment:
new profile so we don't run unnecessary images - including those we cannot build like `open-autonomy-user` which depends on to-be-released versions